chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -972,6 +972,7 @@ integration.upgradeTemplate = The %S plugin for %S is outdated. Reinstall the pl
 integration.mendeleyImport.title          = Missing Mendeley Data
 integration.mendeleyImport.description    = %1$S detected that the document you are citing with contains Mendeley citations. %1$S will be able to manage these citations if you import your Mendeley database.
 integration.mendeleyImport.openImporter   = Open Mendeley Importer...
+integration.citationExplorer.citationsModified = Citations in your document have been modified since Citation Explorer has been opened and %S will not be able to activate them until you reopen this window.
```

Comment:
I know this is from the citation explorer, but so we don't lose track of it, let's 1) make this an .ftl string and 2) make it two sentences (`opened and %S` → `opened. { -app-name } will`). Also, @adomasven, what does "activate" mean here? Is there another verb we can use?